blockchain/state_manager/src/lib.rs

```diff
@@ -375,6 +391,53 @@ where
         })
     }
 
+    fn tipset_executed_message(
+        block_store: &DB,
+        tipset: &Tipset,
+        cid: &Cid,
+        (message_from_address, message_sequence): (&Address, &u64),
```

Comment:
Thought it made sense since instead of passing in a message, I am passing in variables that belong to the message for the purpose of optimizing my clones

Reply:
cool!